bindings/java/src/org/sleuthkit/datamodel/Account.java

```diff
@@ -116,40 +118,25 @@ public int hashCode() {
 		public String toString() {
 			return " displayName=" + this.displayName
 					+ ", typeName=" + this.typeName + ")";
-
-		}
-
-		/**
-		 * Determines if this account type object is equivalent to another
-		 * account type object.
-		 *
-		 * @param that the other type
-		 *
-		 * @return true if it is the same type
-		 */
-		private boolean sameType(Account.Type that) {
-			return this.typeName.equals(that.getTypeName())
-					&& this.displayName.equals(that.getDisplayName());
 		}
 	}
 
 	Account(long account_id, Account.Type accountType, String accountUniqueID) throws TskCoreException {
-
 		this.account_id = account_id;
-
 		this.accountType = accountType;
 		this.accountUniqueID = accountUniqueID;
-
 	}
 
 	/**
-	 * Gets unique string identifier for the account
+	 * Gets unique identifier (assigned by a provider) for the account.
+	 * Example includes an email address.
```

Comment:
I think the notion of provider should be elaborated, if we are going to introduce it.

Reply:
Agreed.  That's why I made the jira to decide to elaborate or pick something else.